src/index.js

```diff
@@ -0,0 +1,162 @@
+    "phone": "(912) 555-4321",
+    "website": "http://johndoe.com",
+    "summary": "A summary of John Doe...",
+    "location": {
+      "address": "2712 Broadway St",
+      "postalCode": "CA 94115",
+      "city": "San Francisco",
+      "countryCode": "US",
+      "region": "California"
+    },
+    "profiles": [{
+      "network": "Twitter",
+      "username": "john",
+      "url": "http://twitter.com/john"
+    }]
+  },
+  "work": [{
+    "company": "Company",
+    "position": "President",
+    "website": "http://company.com",
+    "startDate": "2013-01-01",
+    "endDate": "",
+    "summary": "Description...",
+    "highlights": [
+      "Started the company"
+    ]
+  },
+  {
+    "company": "CompanyB",
+    "position": "VP of Operation",
+    "website": "http://companyB.com",
+    "startDate": "2012-01-01",
+    "endDate": "2013-01-01",
+    "summary": "Some info...",
+    "highlights": [
+      "Created awesomeness"
+    ]
+  }],
+  "volunteer": [{
+    "organization": "Organization",
+    "position": "Volunteer",
+    "website": "http://organization.com/",
+    "startDate": "2012-01-01",
+    "endDate": "2013-01-01",
+    "summary": "Description...",
+    "highlights": [
+      "Awarded 'Volunteer of the Month'"
+    ]
+  }],
+  "education": [{
+    "institution": "University",
+    "area": "Software Development",
+    "studyType": "Bachelor",
+    "startDate": "2011-01-01",
+    "endDate": "2013-01-01",
+    "gpa": "4.0",
+    "courses": [
+      "DB1101 - Basic SQL"
+    ]
+  }],
+  "awards": [{
+    "title": "Award",
+    "date": "2014-11-01",
+    "awarder": "Company",
+    "summary": "There is no spoon."
+  }],
+  "publications": [{
+    "name": "Publication",
+    "publisher": "Company",
+    "releaseDate": "2014-10-01",
+    "website": "http://publication.com",
+    "summary": "Description..."
+  }],
+  "skills": [{
+    "name": "Web Development",
+    "level": "Master",
+    "keywords": [
+      "HTML",
+      "CSS",
+      "Javascript"
+    ]
+  }],
+  "languages": [{
+    "language": "English",
+    "fluency": "Native speaker"
+  }],
+  "interests": [{
+    "name": "Wildlife",
+    "keywords": [
+      "Ferrets",
+      "Unicorns"
+    ]
+  }],
+  "references": [{
+    "name": "Jane Doe",
+    "reference": "Reference..."
+  }]
+
+}
+
+const createWorkItem = (work) => {
+  const highlights = [`<li>${work.summary}</li`];
+
+  for (const highlight of work.highlights) {
+    highlights.push(`<li> ${highlight} </li>`);
+  }
+  console.log(highlights.join(" "));
+  return `
+    <article class="work">
+      <header> 
+        <span> ${work.company? work.company: work.organization} </span>
+        <span> <span>${work.position.toUpperCase()} </span> <span> ${work.startDate} to ${work.endDate? work.endDate : "present"} </span> </span>
+      </header>
+      <div class="description">
+        <ul>
+          ${highlights.join("")}
+        </ul>
+      </div> 
+    </article>
+  `
+}
+
+if (sample.basics) {
```

Comment:
Here, instead of appending template literals to each and every element on your page, you could add the relevant text (like titles) to the page itself and write a function to add the contents of an object to an element:
`function addStuff(element, object) {
// Add data to element
}`

then call the function with the elements and the data:
`addStuff($('header'), sample.basics.name);`